csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -868,6 +941,12 @@ public Settings(bool formatDefaultValues, TypeRegistry typeRegistry) : this(form
             /// </summary>
             /// <param name="formatEnumsAsIntegers"><c>true</c> to format the enums as integers; <c>false</c> to format enums as enum names.</param>
             public Settings WithFormatEnumsAsIntegers(bool formatEnumsAsIntegers) => new Settings(FormatDefaultValues, TypeRegistry, formatEnumsAsIntegers);
+            
+            /// <summary>
+            /// Creates a new <see cref="Settings"/> object with the specified indentation and the current settings.
+            /// </summary>
+            /// <param name="indent"></param>
+            public Settings WithIndentation(string indent = "  ") => new Settings(FormatDefaultValues, TypeRegistry, FormatEnumsAsIntegers, indent);
```

Comment:
the only public api change (other than the WriteValue that I mentioned)

Reply:
And the public properties above.